Octokit/Models/Request/PullRequestReviewCreate.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+using Octokit.Internal;
+using System.Collections.Generic;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Used to filter requests for lists of pull requests.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class PullRequestReviewCreate : RequestParameters
+    {
+        public PullRequestReviewCreate()
+        {
+            Comments = new List<PullRequestReviewCommentCreate>();
+        }
+
+        /// <summary>
+        /// Which PullRequests to get. The default is <see cref="ItemStateFilter.Open"/>
+        /// </summary>
+        public string CommitId { get; set; }
+
+        /// <summary>
+        /// Filter pulls by head user and branch name in the format of "user:ref-name".
+        /// </summary>
+        public string Body { get; set; }
+
+        /// <summary>
+        /// Filter pulls by base branch name.
+        /// </summary>
+        public string Event { get; set; }
```

Comment:
This should be an enum since it has specific allowed values (the same as the Submit action it looks like, so the same enum could be used)